spring-ai-core/src/main/java/org/springframework/ai/chat/prompt/PromptTemplate.java

```diff
@@ -110,15 +114,31 @@ public PromptTemplate(Resource resource, Map<String, Object> model) {
 		}
 	}
 
+	/**
+	 * @deprecated Use {@link #getStructuredOutputConverter()} instead.
+	 */
 	public OutputParser getOutputParser() {
-		return outputParser;
+		return this.outputParser;
 	}
 
+	/**
+	 * @deprecated Use {@link #setStructuredOutputConverter(StructuredOutputConverter)}
+	 * instead.
+	 */
 	public void setOutputParser(OutputParser outputParser) {
 		Objects.requireNonNull(outputParser, "Output Parser can not be null");
 		this.outputParser = outputParser;
 	}
 
+	public StructuredOutputConverter getStructuredOutputConverter() {
```

Comment:
get/setOutputConverter is probably sufficient, three word getters seem to long to me.  thoughts?